src/test/java/org/sagebionetworks/web/unitclient/widget/entity/controller/EntityActionControllerImplTest.java

```diff
@@ -364,6 +364,7 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
 		CallbackMockStubber.invokeCallback().when(mockGWT).scheduleExecution(any(Callback.class), anyInt());
 
 		when(mockPromptModalConfigurationBuilder.buildConfiguration()).thenReturn(mockPromptModalConfiguration);
+		when(mockSynapseJavascriptClient.getChallengeForProject(anyString())).thenReturn(getDoneFuture(new Challenge()));

```

Comment:
Moved success response to the Before (act as the default response for all tests).